### PR TITLE
Pencode aftermath changes

### PR DESCRIPTION
**Pull request by Deantwo**
Adding the changes I discussed with @RemieRichards and @Miauw62 on https://github.com/tgstation/-tg-station/pull/7287 and https://github.com/tgstation/-tg-station/pull/7287.

Just want to see if I can add a changelog for my two PRs maybe.

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -97,3 +97,15 @@
...
+// Used by Paper and PhotoCopier (and PaperBin once a year).
+#define PAPER_DEF_FONT "Verdana"
+#define PAPER_CRAYON_FONT "Comic Sans MS"
+#define PAPER_SIGNFONT "Times New Roman"
+// Used by PDA's Notekeeper.
+#define PDA_NOTE_FONT "Verdana"
+#define PDA_NOTE_SIGNFONT "Times New Roman"
+// Used by NewsCaster and NewsPaper.
+#define NEWS_ARTICLE_FONT "Verdana"
+#define NEWS_ARTICLE_SIGNFONT "Times New Roman"
```
Comment (Miauw62) [inline code/__DEFINES/misc.dm]: These can probably just be condensed down to PAPER_FONT, SIGNFONT and CRAYON_FONT, since they're the same for everything.

**Comment by Miauw62** [comment]
apart from that, looks good :+1:

File: code/__DEFINES/misc.dm (hunk 1 above)
Comment (Deantwo) [inline code/__DEFINES/misc.dm]: Maybe, just though I'd leave it separate if case someone wants to change the font on one of the three things in the future.
But I can make them into three if you want.

**Comment by Iamgoofball** [comment]
>you can now write in comic sans with crayon

i love it

**Comment by Deantwo** [comment]
>>you can now write in comic sans with crayon

>i love it

It was that before I changed anything.

**Comment by Iamgoofball** [comment]
oh

i still love it

**Comment by Deantwo** [comment]
Hope I did that correctly.
And hope it's ok I added it to here.

File: html/changelogs/Deantwo-PRs-7287-7287.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+author: Deantwo
+
+#delete-after: True
```
Comment (optimumtact) [inline html/changelogs/Deantwo-PRs-7287-7287.yml]: You'll want to uncomment this line, looks good apart from that.

**Comment by RemieRichards** [comment]
:+1:

**Pull request merged**